render_delegate/node_graph.cpp

```diff
@@ -679,10 +679,11 @@ AtNode* HdArnoldNodeGraph::ReadMaterialNode(const HdMaterialNode& node, const Co
             } else
                 continue;
         }
-        if (isMaterialx && paramNameStr == "param_shader_file") {
+        if (isMaterialx && paramNameStr.rfind("param_shader_file", 0) == 0) {
```

Comment:
would `TfStringStartsWith(paramNameStr, "param_shader_file")` also be an option ?